be/src/exec/schema_scanner/schema_tables_scanner.cpp

```diff
@@ -111,6 +111,8 @@ Status SchemaTablesScanner::_get_new_table() {
     _db_index++;
     if (nullptr != _param->common_param->wild) {
         table_params.__set_pattern(*(_param->common_param->wild));
+    } else if (nullptr != _param->common_param->table) {
+        table_params.__set_pattern(*(_param->common_param->table));
```

Comment:
pattern is ok? it maybe use for pattern matching, such as `like`

Reply:
It's OK. This pattern is only used for TABLE_NAME="xxx".